R/modules.R

```diff
@@ -192,39 +37,82 @@ is_arg_used <- function(modules, arg) {
 #'   filters in the listed datasets. `NULL` will hide the filter panel,
 #'   and the keyword `'all'` will show filters of all datasets. `datanames` also determines
```

Comment:
For me using double quotes makes it more direct that this is a string
```suggestion
#'   and the keyword `"all"` will show filters of all datasets. `datanames` also determines
```

Reply:
done